src/tests/test_utils.py

```diff
@@ -1,13 +1,27 @@
 from . import base
+from mock import patch
 import utils
 
-class TestBuildercoreUtils(base.BaseCase):
+class TestUtils(base.BaseCase):
     def setUp(self):
         pass
 
     def tearDown(self):
         pass
 
+    def test_git_remote_branches(self):
+        branches = utils.git_remote_branches('https://github.com/elifesciences/builder')
+        self.assertIsInstance(branches, list)
+        self.assertGreaterEqual(len(branches), 1)
+        self.assertIn('master', branches)
+
+    @patch('sys.stderr')
```

Comment:
if we're mocking `sys.stderr` shouldn't we be testing what is captured rather than the return value of `utils.errcho` ?